src/components/DeliverablesTable/index.tsx

```diff
@@ -51,9 +51,13 @@ const DeliverablesTable = ({
   const { activeLocale } = useLocalization();
   const { isAllowed } = useUser();
   const { selectedOrganization } = useOrganization();
-  const { availableProjects: projects, getProjectName } = useProjects();
+  const { acceleratorOrgs } = useAcceleratorOrgs(true);
   const { selectedParticipant } = useParticipants(participantId);
 
+  const acceleratorProjects = useMemo(() => {
```

Comment:
nit: since this is a single line
```
const acceleratorProjects = useMemo(() =>  acceleratorOrgs?.flatMap((org) => org.projects), [acceleratorOrgs]);
```